Assets/HoloToolkit/SpatialMapping/Scripts/TapToPlace.cs

```diff
@@ -44,7 +100,52 @@ void OnSelect()
             }
         }
 
-        // Update is called once per frame.
+        private void CreateAnchor()
+        {
+            // NOTE: It's good practice to ensure your parent hierarchy or root game object does not already have a World Anchor.
+            // You can handle this in a way that works best for your application.
+            // For example: gameObject.transform.root.GetComponent<WorldAnchor>();
+
+            // Add the world anchor component when done moving an object.
+            WorldAnchor anchor = gameObject.AddComponent<WorldAnchor>();
+            if (anchor.isLocated)
+            {
+                SaveAnchor(anchor);
+            }
+            else
+            {
+                anchor.OnTrackingChanged += Anchor_OnTrackingChanged;
+            }
+        }
+
+        private void SaveAnchor(WorldAnchor anchor)
+        {
+            // Save the anchor to persist holograms across sessions.
+            if (anchorStore.Save(SavedAnchorFriendlyName, anchor))
+            {
+                Debug.Log(gameObject.name + " : " + "World anchor saved successfully.");
+            }
+            else
+            {
+                Debug.LogError(gameObject.name + " : " + "World anchor save failed.");
+            }
+        }
+
+        private void Anchor_OnTrackingChanged(WorldAnchor self, bool located)
+        {
+            if (located)
+            {
+                Debug.Log(gameObject.name + " : " + "World anchor located successfully.");
+
+                SaveAnchor(self);
+                self.OnTrackingChanged -= Anchor_OnTrackingChanged;
+            }
+            else
+            {
+                Debug.LogError(gameObject.name + " : " + "World anchor failed to locate.");
```

Comment:
My point was (poorly made) that this event handler won't be removed if the user Taps to Place _before_ this Anchor is located.